plugins/inputs/logparser/README.md

```diff
@@ -104,6 +104,7 @@ You must capture at least one field per line.
   - ts-httpd         ("02/Jan/2006:15:04:05 -0700")
   - ts-epoch         (seconds since unix epoch, may contain decimal)
   - ts-epochnano     (nanoseconds since unix epoch)
+  - ts-syslog        ("Jan 02 15:04:05", parsed time is set to the current year)
```

Comment:
@danielnelson this is the BSD syslog timestamp format (RFC3164) not the syslog RFC5424 timestamp (basically a RFC3339 micro timestamp).
So probably you want to call it `ts-bsd-syslog`, and maybe also add another format for `ts-syslog`.
Just my 2 cents.

Reply:
The timestamp depends on the format (so RFC) chosen.
For example you can configure `RSYSLOG` to use `RSYSLOG_SyslogProtocol23Format` format, which means RFC5424. With this setting `/var/log/*` will contains `RFC3339MICRO` timestamps (as per RFC5424). Without this setting it will default to the old (RFC3164) BSD syslog (and timestamp) format.

Reply:
Regarding localization. RFC5425 with `RFC3339MICRO` timestamps is not affected.
Rather, timestamps of syslog messages following the old RFC3164 are not localized (must be always in english) ([ref](https://tools.ietf.org/html/rfc3164#section-4.1.2)).

Reply:
I think I'll leave this as `ts-syslog` since it pairs with the built in grok pattern `SYSLOGTIMESTAMP` and there isn't any reason to add a special pattern for rfc5424 style timestamps since we already have `ts-rfc3339` and `ts-rfc3339nano`.  We only need a special type here to inject the year.